index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Portfolio</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <header>
+            <nav class="nav_container">
+                <div class="logo">
+                     <img src="images/MyLogo.png" alt="Logo">
+                </div>
+     
+                <div class="toggle">
+                    <img src="images/Union.png" alt="Logo">
+                </div>
+             </nav>
+        </header>
+
+        <main>
+            <div class="main_container">
+               <div class="box1">
+                   I'm Alex Glad to see you!
+               </div>
+               <div class="box2">
+                I'm a software developer! I can help you build a product, 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don't hestiate to contact me.
+               </div>
+               <div class="box3">
+                   LET'S CONNECT
+               </div>
+               <div class="box4">
+                    <div class="socials">
+                        <img src="images/twitter.png" alt="twitter">
+                    </div>
+                    <div class="socials">
+                        <img src="images/Linkedin.png" alt="Linkedin">
+                    </div>
+                    <div class="socials">
+                        <img src="images/m.png" alt="M">
+                    </div>
+                    <div class="socials">
+                        <img src="images/github.png" alt="github">
+                    </div>
+                    <div class="socials">
+                        <img src="images/peace.png" alt="peace">
+                    </div>
+               </div>
```

Comment:
- [ ] Please consider using a `ul` tag to wrap all the social media icons and use `li` tag to wrap each of the social media icons instead of a `div` tag. This is to improve the semantic structure of your page and improve the code structure too. 

- [ ] Also, each icon should be wrapped with an `a` tag which will link to your social media handles. 

See my suggestion below:

```suggestion
               <ul class="box4">
                    <li class="socials">
                       <a href="#">
                         <img src="images/twitter.png" alt="twitter">
                        </a>
                    </li>
                    <li class="socials">
                      <a href="#">
                        <img src="images/Linkedin.png" alt="Linkedin">
                      </a>
                    </li>
                    <li class="socials">
                      <a href="#">
                        <img src="images/m.png" alt="M">
                      </a>
                    </li>
                    <li class="socials">
                      <a href="#">
                        <img src="images/github.png" alt="github">
                      </a>
                    </li>
                    <li class="socials">
                      <a href="#">
                        <img src="images/peace.png" alt="peace">
                      </a>
                    </li>
               </ul>
```